### ISSUE TITLE
Drop netstandard 2.0

### ISSUE THREAD

**Issue by patriksvensson** [spectreconsole/spectre.console#1441]
---
Please upvote :+1: this issue if you are interested in it.

**Comment by PrzemyslawKlys** [comment]
I would like you to not drop it if possible. PowerShell 5.1 depends on it. If you want Spectre.Console still work there - it's the only way to preserve it. It's similar to:
- https://github.com/spectreconsole/spectre.console/issues/1527

**Comment by BlazeFace** [comment]
Wouldn't it work in AOT mode in those cases, since you would not be framework dependent while also not having to ship the runtime with each app?

**Comment by PrzemyslawKlys** [comment]
No. That's not how PowerShell modules calls it. They call directly DLL, so while your app with AOT may work in PowerSheell console, it won't work if you have PowerShell module which calls SpecteConsole methods. 

And to be honest I am not 100% sure net standard 2.0 will help. I still use net framework 4.7 in https://github.com/EvotecIT/ConsoleMonster as sometimes for whatever reason the dependency hell kicks in and things just don't work. The other project https://github.com/ShaunLawrie/PwshSpectreConsole calls netstandard2.0 directly https://github.com/ShaunLawrie/PwshSpectreConsole/blob/df3a470a6ffe8b10739b6a00b3149e7fca343c58/PwshSpectreConsole/PwshSpectreConsole.psd1#L57C1-L61C93 

So this may work for Spectre.Console, but I know that in other cases I have to resort to 4.7.2 in PowerShell 5.1 for it to work, as it starts throwing errors for whatever reason otherwise. 

So if you can please preserver net standard 2.0, and hopefully it can be brought back for Spectre.Console.ImageSharp, because without it, the projects in PowerShell 5.1 are dead. I know I can't make you to retain 4.7.2 as it was dropped a while ago Net Standard 2.0 is the only thing that keeps PS 5.1 alive, and Microsoft intended to keep it till 2027 at least. Until Microsoft pushes PowerShell 7, as default to Windows, majority of companies will stay on PS 5.1

**Comment by patriksvensson** [comment]
Postponed to 2.0
